manual/src/html_processing/scss/manual.scss

```diff
@@ -307,47 +307,46 @@ div.caml-example.toplevel div.caml-input::before{
 span.number{
     padding-right: 1ex;
 }
-span.c004, span.c005, span.c007 {
+span.machine, span.syntax-token {
 	font-family: $font-mono;
 }
-span.c003, span.c005 {
+span.syntax-token {
 	color: rgba(91, 33, 6, 0.87);
 }
-span.c002{
+span.ocamlprompt{
     color:#888
 }
-span.c006{
+span.font-bold.machine{
     font-weight:700;
     color:#564233;
-    font-family: $font-mono;
 }
-.c008 {
+.osvariant {
 	font-family: $font-sans;
 }
-span.c010 {
+span.font-it {
 	font-style: italic;
 }
 span.authors{
     font-style:italic;
     background-color:inherit
 }
-span.c011 {
+span.nonterminal {
 	font-style: italic;
 }
-.c012 {
-	font-style: italic;
+.font-sl {
+	font-style: oblique;
```

Comment:
In the PDF version, "c012" actually had italic font, not slanted. But again, this doesn't matter with the current web font.

Reply:
This is more a mismatch on the diff, I generalized the `span.font-it` rule to `.font-it`.